tests/encode.cpp

```diff
@@ -159,6 +159,15 @@ int main(int argc, char** argv)
                 fwrite(MVBuffer.data, MVBuffer.bufferSize, 1, MVFp);
             }
 #endif
+            if (status == ENCODE_BUFFER_TOO_SMALL) {
+                maxOutSize = (maxOutSize * 3) / 2;
+                free(outputBuffer.data);
```

Comment:
yes, and you do not need set "outputBuffer.data = NULL" since you will allocate the data later. Please check the outputBuffer, please make sure you memset the outputBuffer in main(), else free(outputBuffer.data) will crash your programe.